src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const basePrice = 40;
+const threeDays = 3;
+const threeDaysDiscount = 20;
+const sevenDays = 7;
+const sevenDaysDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < threeDays) {
+    return basePrice * days;
```

Comment:
The use of `basePrice * days` directly in the return statement is clear, but to improve maintainability and avoid repeating this calculation, you could store the result in a variable like `totalCost` at the beginning of the function and use that variable in your return statements.